### PR TITLE
Edit media

### PR DESCRIPTION
**Pull request by derpdead**
# Description

- added business context for editing resource
- refactor of privileges
- naming conventions improvement
- grid bug fixes

Issue # (#411 )

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Unit Tests
- [ ] e2e Test

# Checklist:

- [x] I have read the contribution requirements and fulfil them.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by dawidstrix** [comment]
Build 119 Successful

File: modules/@ergonode/channels/src/components/Forms/ExportDetailsForm.vue [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        TableRowCell,
+        TableRowInfoCell,
+    },
+    computed: {
+        ...mapState('channels', {
+            export_details: state => state.export_details,
+        }),
+        rows() {
+            return Object.keys(this.export_details).reduce((acc, current) => {
+                const tmpArray = acc;
```
Comment (tomaszstrozik) [inline modules/@ergonode/channels/src/components/Forms/ExportDetailsForm.vue:55]: Why are you creating extra variable each time in loop? Is it some eslint rule to meet (like "don't mutate function input")?

File: modules/@ergonode/channels/src/store/channels/actions.js [hunk 2]
```diff
@@ -83,6 +83,7 @@ export default {
         } = rootState.authentication.user;
 
         return this.app.$axios.$get(`${userLanguageCode}/channels/${channel_id}/exports/${export_id}`).then((data) => {
+            console.log(data);
```
Comment (tomaszstrozik) [inline modules/@ergonode/channels/src/store/channels/actions.js:86]: redundant console.log I think

File: modules/@ergonode/channels/src/components/Forms/ExportDetailsForm.vue (hunk 1 above)
Comment (derpdead) [reply modules/@ergonode/channels/src/components/Forms/ExportDetailsForm.vue:55]: Yeah, immutability rule

**Pull request merged**